app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -51,71 +57,73 @@
                         <Button @type='secondary' {{on 'click' dd.actions.close}}>
                             {{t 'general.cancel'}}
                         </Button>
-                        <Button @type='primary' {{on 'click' this.applyColumnSelection}}>
+                        <Button @type='primary' {{on 'click' this.applyColumnSelection}} {{on 'click' dd.actions.close}}>
```

Comment:
Just for future reference, if you want to have a click even do multiple actions in sequence, you can use either the `pipe` or `queue` helper like 
`{{on 'click' (queue this.firstAction this.secondAction)}}`

Docs for those can be found here: https://github.com/DockYard/ember-composable-helpers?tab=readme-ov-file#queue